test/client/fixtures/sandbox/cookie-test.js

```diff
@@ -228,3 +194,27 @@ test('correct work with cookie with empty key (GH-899)', function () {
 
     transport.queuedAsyncServiceMsg = savedQueuedAsyncServiceMsg;
 });
+
+if (!browserUtils.isIOS) {
```

Comment:
Add comment for this - 'Browsers on iOS platform doesn't support `beforeunload` event. We cann't use the `pagehide` event for cookie synchronization. Request for a new page will be handled earlier than sync xhr request was sent from `pagehide` event handler.'